code/modules/surgery/organs/organ_movement.dm

```diff
@@ -193,7 +193,7 @@
 
 	// The true movement is here
 	moveToNullspace()
-	bodypart_owner.contents -= src
+	bodypart_owner?.contents -= src
```

Comment:
Funnily enough I came across this line of code myself recently... why are we manually removing something from the contents list anyways? 🤔

Reply:
I don't know 🙂

Reply:
What happens when you just remove the line?

Reply:
if the `loc` of `src` is `bodypart_owner` then the `loc` of `src` will be set to `null` by this line.
However, `moveToNullspace()` is called directly before this. So this line is essentially a no-op.

Reply:
Thinking about it more I think this change was an artifact of an earlier version where I was registering to `COMSIG_ORGAN_REMOVED` and not `COMSIG_ORGAN_REMOVED_SURGICALLY`
It would runtime because I would delete the body part upon receiving the signal, deleting the bodypart would remove it from its owner and null `bodypart_owner`, and then it would reach here afterwards